packages/operator/pkg/webserver/routes/v1/packaging/model_packaging_validation_test.go

```diff
@@ -465,3 +468,43 @@ func (s *ModelPackagingValidationSuite) TestMpResourcesValidation() {
 	s.g.Expect(errorMessage).Should(ContainSubstring(
 		"validation of gpu limit is failed: quantities must match the regular expression"))
 }
+
+func (s *ModelPackagingValidationSuite) TestOutputConnection() {
+
+	validator := pack_route.NewMpValidator(s.mpRepository, s.connRepository)
+
+	testMpOutConnDefault := testOutConnDefault
+	testMpOutConn := testOutConn
+	testMpOutConnNotFound := testOutConnNotFound
+
+	mp := &packaging.ModelPackaging{
+		Spec: packaging.ModelPackagingSpec{},
+	}
+
+	// If configuration output connection is not set then user must specify it as ModelTraining parameter
+	viper.Set(packaging_config.OutputConnectionName, nil)
+	err := validator.ValidateAndSetDefaults(mp)
+	s.g.Expect(err).To(HaveOccurred())
+	s.g.Expect(err.Error()).To(ContainSubstring(fmt.Sprintf(validation.EmptyValueStringError, "OutputConnection")))
+
+	// If configuration output connection is set and user has not passed output connection as training
+	// parameter then output connection value from configuration will be used as default
+	viper.Set(packaging_config.OutputConnectionName, testMpOutConnDefault)
```

Comment:
Reset the packaging_config.OutputConnectionName viper value to the nil in the test tear down.